app/assets/stylesheets/application.scss

```diff
@@ -103,6 +106,35 @@ div.grader-config {
     padding: 10px;
 }
 
+.review-navbar .navbar-header { width: 100%; vertical-align: middle; }
+.review-navbar h3 {
+   margin-top: 10px;
+   vertical-align: middle; 
+   width: 100%;
+   line-height: unset;
+   padding: 0;
+}
+
+
+@media(min-width: 768px) {
+   nav.review-navbar {
+      position: sticky;
```

Comment:
??  The review-navbar only appears on Codereview pages, and this CSS works *much* better than the buggy jQuery equivalents (which don't compose, and don't allow multiple scrolling triggers on the page; it's a longstanding, multiple-times-reported issue that jQuery refuses to fix for some reason).